README.md

```diff
@@ -242,6 +242,16 @@ In MVC views `<feature>` tags can be used to conditionally render content based
 </feature>
 ```
 
+The tag helper can encompass multiple features. By listing all feature names, separated by commas, and specifing the `requirement` in the example below, the content is only rendered if `FeatureX` and `FeatureY` are both enabled.

```

Comment:
As a reader, I think negate concept makes sense to come before requirement concept.